starter-webapp-core/src/test/java/org/camunda/bpm/spring/boot/starter/webapp/filter/LazyDelegateFilterTest.java

```diff
@@ -17,12 +17,7 @@
 package org.camunda.bpm.spring.boot.starter.webapp.filter;
 
 import static org.junit.Assert.assertSame;
-import static org.mockito.Mockito.doReturn;
-import static org.mockito.Mockito.inOrder;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.spy;
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.*;
```

Comment:
Hi @tobiasschaefer, 

According to the project guidelines, we don't allow wildcard imports. So I would restore the separate imports and just add the `org.mockito.Mockito.never` static import:
```suggestion
import static org.mockito.Mockito.doReturn;
import static org.mockito.Mockito.inOrder;
import static org.mockito.Mockito.mock;
import static org.mockito.Mockito.never;
import static org.mockito.Mockito.spy;
import static org.mockito.Mockito.times;
import static org.mockito.Mockito.verify;
```

Best,
Nikola